packages/form-js-viewer/src/render/components/form-fields/Taglist.js

```diff
@@ -120,13 +119,13 @@ export default function Taglist(props) {
       id={ prefixId(id, formId) } />
     <div class={ classNames('fjs-taglist', { 'disabled': disabled }) }>
       {!disabled && loadState === LOAD_STATES.LOADED &&
-        selectedValues.map((sv) => {
+        values.map((v) => {
           return (
             <div class="fjs-taglist-tag" onMouseDown={ (e) => e.preventDefault() }>
               <span class="fjs-taglist-tag-label">
-                {sv.label}
+                {valueToOptionMap[v] ? valueToOptionMap[v].label : `unexpected value{${v}}`}
```

Comment:
Do we have a test case for this? If not, let's add it.

Reply:
Well this is a bit of a strange one, but basically it's a state you shouldn't be able to get into unless there is a code issue. I will change this actually, we can instead make the sanitation bullet proof and check if the label is present there. I think that makes more sense.

Reply:
Actually I've removed this entirely to match behavior with all our other controls. If no label is defined we just don't display a label. Perhaps we may want to do something else in the future, but let's be consistent before we set a behavior in stone. I've raised it as a follow up: https://github.com/bpmn-io/form-js/issues/296